From f6e751c60c8fdfd3d8f25e69f4ac57189a5a7edb Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Tue, 2 Aug 2016 23:21:35 +0200
Subject: [PATCH 12/17] sepolicy: recovery: Fix the volume manager blkid.tab
 denial

 * denied  { link } for  pid=573 comm="recovery"
    name="vold_blkid.tab" dev="tmpfs" ino=20425
    scontext=u:r:recovery:s0 tcontext=u:object_r:tmpfs:s0
    tclass=file

Change-Id: I0b3e47dd00c5a32261691f51838a8d9af9778faa

# Conflicts:
#	common/private/recovery.te
---
 common/private/recovery.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index bd1d9d0..9efdbf4 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -35,4 +35,6 @@ allow recovery self:netlink_kobject_uevent_socket create_socket_perms_no_ioctl;
 
 # Set time
 allow recovery self:capability sys_time;
+
+allow recovery tmpfs:file link;
 ')
-- 
2.17.1

